src/core.js

```diff
@@ -65,6 +65,11 @@ register([
     require('./locale-en-us')
 ]);
 
+// locales that are present in the window should be loaded
+if(window.PlotlyLocales && window.PlotlyLocales.length) {
+    register(window.PlotlyLocales);
```

Comment:
Maybe we could after this line have:

```js
delete window.PlotlyLocales;
```

to de-pollute the window namespaces after we're done registering the locales.

Reply:
Done in f169380281377e3af40bb0772112d8ad9084f092

Reply:
Since it is a temporary variable, it may be better to add an underscore i.e. `window._PlotlyLocales`.

Reply:
Thanks for the suggestion! I'll let @etpinard choose between the different options (see also https://github.com/plotly/plotly.js/pull/4453#issuecomment-569838778)

Reply:
Wondering using [sessionStorage](https://developer.mozilla.org/en-US/docs/Web/API/Window/sessionStorage) may be a possibility?